### ISSUE TITLE
FTB App Import Issue with Permafrost - Eyes of the Storm

### ISSUE THREAD

**Issue by MonfGeiger** [MultiMC/Launcher#5113]
### Operating System

Windows

### Description of bug

MultiMC version 0.7.0-3732. MMC refuses to import PermaFrost - Eyes of the Storm. Basically i remember importing it an age & half ago, figured i'd check on it in MultiMC, opened its instance menu and saw that the loader mods section was blank. so i deleted it and reimported it, and it only imports the config folder now, nothing else. I even downloaded a fresh MultiMC setup and tried that, and the same issue persisted. I have no clue why this is happening beyond something in this edition of MultiMC must be on the fritz. the only thing i haven't tried but will be doing at the time of this post is uninstalling the FTB App and reinstalling it, seeing if that does anything special for my issue, if not, then ill refer to those that know more than me. Here is my MMC log if that helps any
[MultiMC-0.log](https://github.com/MultiMC/Launcher/files/11096078/MultiMC-0.log)

### Steps to reproduce

Download a fresh instance of Permafrost - Eyes of the Storm from the FTB App (i'm using 5.6.2), open up MultiMC, go to FTB App Import, import it, and open the new MultiMC instance folder.

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.

**Comment by phit** [comment]
> 15.121 W Failed to copy "config/fancymenu/Instances - Shortcut.lnk"

**Comment by MonfGeiger** [comment]
wanted to put this out there, uninstalling and reinstalling the FTB App did absolutely nothing in any regard to importing the Permafrost modpack, same issue as before

**Comment by FeedTheChunk** [comment]
Just copy the files manually.  It's probably do to the files being in your users directory and access is being blocked by Defender or similar.

**Comment by MonfGeiger** [comment]
if it was that simple to fix, then i wouldnt have made this issue tracker, but i tried it anyway, and to no ones surprise, it failed. Here is a bundle of logs from both the permafrost instance, the popup crash log that MultiMC showed, and the MultiMC-0 log file in a zip package (no password needed)
[Logs Bundle.zip](https://github.com/MultiMC/Launcher/files/11128398/Logs.Bundle.zip)

If anything more can be puzzled out of these logs that'd help, i'd be mighty appreciative of it, because copy/paste didnt work here

**Comment by phit** [comment]
Well that game crash you had now has nothing to do with the copy problem, here's the fix for that, the bug here is that MultiMC can't copy a `lnk` file aka shortcut.

---

Forge 34.1.27 to 36.2.24 (Minecraft 1.16.3-1.16.5) are broken with Java 8 Update 321 or newer.
If you are on 1.16.5 try upgrading your instances Forge version. If this isn't an option for you, you need an older Java 8 version, e.g. 8u312 you can find downloads here <https://www.azul.com/downloads/?version=java-8-lts&os=windows&architecture=x86-64-bit&package=jre&show-old-builds=true#download-openjdk>

the crash will contain this
```
Caused by: java.lang.NoSuchMethodError: 'void sun.security.util.ManifestEntryVerifier.<init>(java.util.jar.Manifest)'
```

**Comment by MonfGeiger** [comment]
I have 8u351 as the latest java 8 installed with args to optimize it "from old server i played in), with jdk-17 and jdk 19 installed for the versions of minecraft that need them. the default java MMC points to is java 8, but for the instances that need the higher ones, i adjust their instance settings to point to the higher java and ensure the args settting is enabled but left blank so there's no setting conflict. ill download the older one and see how that works tho

**Comment by MonfGeiger** [comment]
huh......well, that seems to have done it. used the zip archive of the zulu 8u312, pointed permafrost to it, loaded it up, and its running just fine. But...i already have 3 java runtimes on my computer, tacking on a fourth one is a bit excessive. i see the modpack devs have a github issue tracker, so ill move the next part of my inquiry to them and see what (and IF anything) they say. But i figured that instead of copying the lnk/shortcut file, MMC would follow it and resolve whatever paths it lead to

any chance a future MMC update would fix this blunder, or is this the one thing ill have to copy/paste from now on even during a modpack update?

**Comment by FeedTheChunk** [comment]
It's a well know issue with Forge not supporting the "newer" versions of Java 8 like Phil mentioned, It's a huge PITA as you see, now you have to have so many versions of Java installed.  Luckily it only affects those versions of Forge that Phil mentioned in his resolution.

**Comment by MonfGeiger** [comment]
fuckin.......now i wish forge wasn't so janky in this regard. guess ill be haivng multiple versions of java either way, installed or portable. I guess my issue with getting it running is solved, but idk if that means this bug stays open or not in this case

**Comment by phit** [comment]
the bug has nothing to do with your java version problems

**Comment by phit** [comment]
Duplicate of https://github.com/MultiMC/Launcher/issues/3236
